### PR TITLE
Production mode

### PR DESCRIPTION
**Pull request by mikecroft**
* Slight reorg to arrange admin console enhancements into their own section
* Fixed case on asadmin recorder page
* Added intro page for the new section

Preview here:
https://mikecroft.gitbooks.io/payara-server-documentation-fork/content/v/production-mode/documentation/extended-documentation/admin-console/environment-warning.html

### CODE REVIEW

File: documentation/extended-documentation/admin-console/environment-warning.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+= Admin Console Environment Warning
+
+_Since Payara Server 172_
+
+When working with multiple shared environments, it is common for restrictions to be in place on what modifications can be allowed for a given domain. For example, the test environment cannot deviate significantly from the target production environment without potentially invalidating the test.
+
+One way to ensure no accidental changes are made would be to revoke access to the admin console, but this has the downside of preventing engineers from seeing the current settings in the event that a bug is found.
+
+As an alternative, Payara Server now ships with a configurable warning bar to ensure any user can see a warning message in while viewing the console:
```
Comment (michaelranaldo) [inline documentation/extended-documentation/admin-console/environment-warning.adoc]: "...will see a warning message while viewing the console."

File: documentation/extended-documentation/admin-console/environment-warning.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+|====
+|Option
+|Type
+|Description
+|Default
+|Mandatory
+
+|enabled
+|boolean
+|If it is set to true, the top warning banner is showed, otherwise it is not.
```
Comment (michaelranaldo) [inline documentation/extended-documentation/admin-console/environment-warning.adoc]: shown? Unsure myself

Comment (OndrejM) [inline documentation/extended-documentation/admin-console/environment-warning.adoc]: I always use "displayed" to avoid a mistake ;-)

**Comment by OndrejM** [comment]
I tend to place documentation about improvements/additions in existing GlassFish features into the core section. As the Admin console itself is also available in GlassFish, I would add the docs of the Admin console into the core section, and place Payara-specific additions under a special "Payara section".

However, maybe we should abandon the idea of separating "Core" and "Extended" docs completely. What do you think? The idea of Core and extended is clear, but it's not clear where improvements of existing GlassFish functionality fits. Unless we really start documenting common GlassFish and Payara functionality (and essentially duplicating or complementing GlassFish docs), we should probably put all our docs into the Extended docs, or just merge both sections into a single "Payara Server" one?

File: SUMMARY.adoc [hunk 3]
```diff
@@ -33,6 +33,9 @@
 *** link:documentation/core-documentation/rest-api/definitions.adoc[Definitions]
 ** link:documentation/core-documentation/javamail.adoc[JavaMail]
 * link:documentation/extended-documentation/extended-documentation.adoc[Extended Documentation]
+** link:documentation/extended-documentation/admin-console/admin-console.adoc[Admin Console Enhancements]
+*** link:documentation/extended-documentation/admin-console/asadmin-recorder.adoc[Asadmin Recorder]
```
Comment (OndrejM) [inline SUMMARY.adoc]: Be careful with moving pages, it breaks old URLs. We might use the [redirect plugin](https://plugins.gitbook.com/plugin/bulk-redirect) to redirect old links.

Comment (OndrejM) [inline SUMMARY.adoc]: I created "PYSVC-412: Create a job in Jenkins to find broken URLs" to track this.

Comment (mikecroft) [inline SUMMARY.adoc]: Verified it only appears in the summary

**Pull request merged**